Prox/Prox/Models/PlaceProviders.swift

```diff
@@ -145,9 +145,8 @@ extension SourcePlaceProvider {
         var ids = [String]()
         for category in value {
             guard let name = category["text"],
-                let id = category["id"] else {
-                    print("lol unable to retrieve category from firebase data for place")
```

Comment:
I did because I was thinking that a place won't necessarily have a category, but looking at this again, this is only logged if a place _does_ have a category but the necessary fields are missing. I'll add it back.